src/Conference/AddConference.jsx

```diff
@@ -6,12 +6,12 @@ import {
   StyledCard
 } from "../StyledFormComponents";
 
-const AddConference = () => {
+const AddConference = sessionToken => {
```

Comment:
This arguement is actually `props` which you have just called sessionToken, do you not want to deconstruct it to `({ sessionToken })` ?

Reply:
fixed!